data/src/generate_tsv_summary.py

```diff
@@ -11,7 +11,31 @@
 
 
 def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[5]
+    return ele[3] + ele[7]
+
+
+def _handle_modifier(
+    language: Dict[str, Any],
+    file_path: str,
+    modifier: str,
+) -> str:
+    modifiers = language.get(modifier, {})
+    key = file_path[file_path.index("_") + 1:file_path.rindex("_phone")]
+    if "_" in key:
+        if modifier == "script":
+            key = key[: key.index("_")]
+        elif modifier == "dialect":
+            key = key[key.index("_") + 1:]
+    return modifiers.get(key, "")
+
+
+def _handle_transcription_level(file_path: str) -> str:
+    trans = file_path[
+        file_path.index("phone"):file_path.index(".")
+    ].capitalize()
+    if "_" in trans:
+        trans = trans[: trans.index("_")]
+    return trans
 
 
 def _handle_wiki_name(
```

Comment:
We can dump this entire function now right? However, you'll want to integrate the
```
            if "|" in values:
                values = values.replace(" |", ",")
```
functionality somewhere else since our English dialects are stored like so:
```
        "dialect": {
            "us": "US | General American",
            "uk": "UK | Received Pronunciation"
        },
```
And that "|" is throwing off the README table for English.

Reply:
Good catch. I guess replace " | " with ", " maybe?

Reply:
+1